### PR TITLE
PLS-926 Label-song ES mapping

### PR DESCRIPTION
**Pull request by dkimdkimdkim**
- [x] tested

### CODE REVIEW

File: templates/label_song_template.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                "enabled": false
+            },
+            "_parent": {
+                "type": "label"
+            },
+            "properties": {
+                "song_id": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "store": "no"
```
Comment (CarlosMecha) [inline templates/label_song_template.json]: `store`: `yes`

File: templates/label_song_template.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                "enabled": false
+            },
+            "_all": {
+                "enabled": false
+            },
+            "properties": {
+                "label_id": {
+                    "type": "long",
+                    "index": "not_analyzed",
+                    "store": "no"
```
Comment (CarlosMecha) [inline templates/label_song_template.json]: `store: yes`

File: templates/label_song_template.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                },
+                "label_name": {
+                    "type": "string",
+                    "index": "not_analyzed",
```
Comment (CarlosMecha) [inline templates/label_song_template.json]: `"index": "analyzed"`

File: templates/label_release_template.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            "properties": {
+                "label_id": {
+                    "type": "long",
+                    "index": "not_analyzed",
+                    "store": "no"
+                },
+                "label_name": {
+                    "type": "string",
+                    "index": "not_analyzed",
```
Comment (CarlosMecha) [inline templates/label_release_template.json]: `"index": "analyzed"`

File: templates/label_release_template.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            "_source": {
+                "enabled": false
+            },
+            "_all": {
+                "enabled": false
+            },
+            "properties": {
+                "label_id": {
+                    "type": "long",
+                    "index": "not_analyzed",
```
Comment (CarlosMecha) [inline templates/label_release_template.json]: `"store": "yes"`

File: templates/label_release_template.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+                "enabled": false
+            },
+            "_parent": {
+                "type": "label"
+            },
+            "properties": {
+                "release_id": {
+                    "type": "string",
+                    "index": "not_analyzed",
+                    "store": "no"
```
Comment (CarlosMecha) [inline templates/label_release_template.json]: `"store": "yes"`

File: templates/label_song_template.json (hunk 2 above)
Comment (dkimdkimdkim) [inline templates/label_song_template.json]: done

File: templates/label_song_template.json (hunk 3 above)
Comment (dkimdkimdkim) [inline templates/label_song_template.json]: done

File: templates/label_song_template.json (hunk 1 above)
Comment (dkimdkimdkim) [inline templates/label_song_template.json]: done

File: templates/label_song_template.json [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+                    "store": "yes"
+                },
+                "song_title": {
+                    "type": "string",
+                    "index": "analyzed",
+                    "store": "no"
+                },
+                "song_ranking": {
```
Comment (CarlosMecha) [inline templates/label_song_template.json]: We don't need the song ranking, that's just applied to `artists`, thanks!

Comment (dkimdkimdkim) [inline templates/label_song_template.json]: done

**Pull request merged**